### PR TITLE
Convert Accelerometer Driver from gdrive to md

### PR DESCRIPTION
**Pull request by hsorathia**

### CODE REVIEW

File: documentation/design_docs/L2/Accelerometer Driver.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+# Background
+
+The MMA8452Q accelerometer is a device that communicates using the I2C communication protocol. It can be read from and written to through the I2C communication line with the SJTwo microcontroller.
+
+The accelerometer driver requires an I2C API to function.
+
+# Overview (all)
+
+The purpose of the accelerometer driver is to allow the SJTwo microcontroller to communicate with the MM8452QA accelerometer. At the base, the driver obtains data of the X, Y, and Z axes independently. In addition, the driver includes processing sensor data to determine device pitch and roll.
```
Comment (kammce) [inline documentation/design_docs/L2/Accelerometer Driver.md]:
```suggestion
The purpose of the accelerometer driver is to allow any microcontroller to communicate with an accelerometer like the MM8452QA. At the base, the driver obtains data of the X, Y, and Z axes independently. In addition, the driver includes processing sensor data to determine device pitch and roll.
```

File: documentation/design_docs/L2/Accelerometer Driver.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+N/A
+
+# Testing Plan
+
+N/A
+
+## Unit Testing Scheme
+
+N/A
```
Comment (kammce) [inline documentation/design_docs/L2/Accelerometer Driver.md]:
```suggestion
<missing>
```

File: documentation/design_docs/L2/Accelerometer Driver.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+# Future Advancements (optional)
+
+N/A
+
+# Testing Plan
+
+N/A
+
```
Comment (kammce) [inline documentation/design_docs/L2/Accelerometer Driver.md]:
```suggestion
```

Comment (kammce) [reply documentation/design_docs/L2/Accelerometer Driver.md]: Testing plan should have nothing in it, usually.

File: documentation/design_docs/L2/Accelerometer Driver.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+## Configuring the Sensor
+
+The full scale range of the sensor can be configured through the “CFG” register. The user is given the option of setting the full scale range to 2G, 4G, or 8G.
+
+# Caveats
+
+N/A
+
+# Future Advancements (optional)
```
Comment (kammce) [inline documentation/design_docs/L2/Accelerometer Driver.md]:
```suggestion
# Future Advancements
```
Remove the () parts after each title.

File: documentation/design_docs/L2/Accelerometer Driver.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+I2C class. This function writes the appropriate register address into the
+acceleration sensor and then reads back the next 16 bits from the slave.
+Since data from the sensor is only 12 bits, the value returned from the sensor
+is divided by 16 to remove this offset and maintain the polarity. Furthermore,
+the pitch and roll functions fetch relevant axis values through the get
+functions for calculations.
+
+## Configuring the Sensor
+
+The full scale range of the sensor can be configured through the “CFG” register.
```
Comment (kammce) [inline documentation/design_docs/L2/Accelerometer Driver.md]: Please change he qoutes here from the fancy quotes to normal ". This is something that happens when you copy and paste from Google Docs or Microsoft word. Please do this everywhere. There are a few areas above as well.

**Comment by kammce** [comment]
Please rebase your local branch against master.

File: documentation/design_docs/L2/accelerometer.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
+# Accelerometer Driver
```
Comment (kammce) [inline documentation/design_docs/L2/accelerometer.md]:
```suggestion
# Accelerometer Interface with Implementation
```

File: documentation/design_docs/L2/Accelerometer Driver.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
+# Accelerometer Driver
```
Comment (kammce) [inline documentation/design_docs/L2/Accelerometer Driver.md]: Delete this file.

**Comment by hsorathia** [comment]
I went through the update/pulled the new commits, did a git rebase -i master to squash those new commits that got added into my original commit, and pushed everything as one commit.

**Pull request closed without merging**